gamemodes/terrortown/gamemode/server/sv_addonchecker.lua

```diff
@@ -184,25 +184,40 @@ addonChecker.curatedList = {
 	},
 	["1729451064"] = { -- TTT - More Melee Weapons by Solrob
 		reason = "Overwrites stock TTT2 crowbar which chauses problems with doors and ttt_map_settings entity.",
```

Comment:
not your fault, but could you fix this typo while you're at it? Thanks!

Reply:
fixed, had to remerge from origin, hope it didn't break anyting